src/api/channel/by_gop.rs

```diff
@@ -52,7 +52,7 @@ impl<T: Pixel> SceneChange<T> {
     let seq = Arc::new(Sequence::new(enc));
 
     let detector = SceneChangeDetector::new(
-      *enc,
+      enc.clone(),
```

Comment:
I don't understand why this switched to being a clone().

Reply:
Unfortunately there needed to be a field added for the film grain settings which is a Vec, which prevents it from being copy-able.